plane_mcp/journey/cache.py

```
@@ -0,0 +1,110 @@
+import os
+import json
+import time
+import tempfile
+import logging
+
+def get_cached_workspace_context(cache_ttl_seconds: int = 300) -> dict:    
+    from plane_mcp.client import get_plane_client_context
+    try:
+        ctx = get_plane_client_context()
+        ws = ctx.workspace_slug or "default"
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)